dockerArgs

```diff
@@ -0,0 +1,24 @@
+ARG ARROW_CPP_VERSION=0.14.1
+ARG CC_VERSION=7
+ARG CFFI_VERSION=1.11.5
+ARG CMAKE_VERSION=3.14
+ARG CXX_VERSION=7
+ARG CYTHON_VERSION=0.29.*
+ARG DASK_VERSION=">2"
+ARG DASK_XGBOOST_CONDA_VERSION_SPEC=0.2*
+ARG DISTRIBUTED_VERSION=">2"
+ARG FAISSGPU_VERSION=1.5.0
+ARG HASH_JOIN=ON
+ARG IPYTHON_VERSION=7.3*
+ARG NCCL_CONDA_VERSION=">2"
+ARG NUMBA_VERSION=0.41
```

Comment:
This needs to be updated to match the new numba version of 0.44 along with these other changes https://github.com/rapidsai/gpuci-build-environment/pull/35/files